spec/gitx/cli/base_command_spec.rb

```diff
@@ -21,9 +21,9 @@
   describe 'with custom .gitx.yml config file' do
     let(:config) do
       {
-  'aggregate_branches' => %w( foo bar ),
-  'reserved_branches' => %w( baz qux ),
-  'taggable_branches' => %w( quux corge )
+        'aggregate_branches' => %w( foo bar ),
+        'reserved_branches' => %w( baz qux ),
+        'taggable_branches' => %w( quux corge )
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.